playbooks/byo/openshift-cluster/upgrades/v3_5/upgrade_control_plane.yml

```diff
@@ -104,3 +104,5 @@
 - include: ../../../../common/openshift-cluster/upgrades/upgrade_control_plane.yml
 
 - include: ../../../../common/openshift-cluster/upgrades/post_control_plane.yml
+
+- inlcude: ../../../../common/openshift-cluster/upgrades/v3_5/storage_upgrade.yml
```

Comment:
I don't care about nodes with this, since I'm talking with master which is responsible for storing objects. So in the situation you're talking about this should be ok.